src/SpecificDissipationRateEquationSystem.C

```diff
@@ -323,6 +328,13 @@ SpecificDissipationRateEquationSystem::register_inflow_bc(
   std::vector<double> userSpec(1);
   userSpec[0] = sdr.specDissRate_;
 
+  std::printf("Open file for writing sdrFS\n");
```

Comment:
Would this be better if it's written to the Nalu-Wind output file?

`NaluEnv::self().naluOutputP0() << "SDR EqnSys Inlet omega value = " <<  userSpec[0] << std::endl;`

Reply:
Transfer of freestream inflow values to  node_kernels via file write/read has been replaced by adding them to Realm class.